src/managers/world-manager.js

```diff
@@ -144,7 +143,7 @@ module.exports = function (options) {
 
             function getAndRestoreLatestRun(world) {
                 var currentWorldId = world.id;
-                var strategy = buildStrategy(currentWorldId, dtd);
+                var strategy = buildStrategy(currentWorldId, model, dtd);
```

Comment:
The model can be passed in through the original options so maybe this should be `model || options.model`?